kroxylicious-migrations/src/main/resources/META-INF/rewrite/rewrite.yaml

```diff
@@ -7,10 +7,19 @@
 ---
 type: specs.openrewrite.org/v1beta/recipe
 name: io.kroxylicious.rewrite.maven.EncryptionModuleToRecordEncryption
-displayName: Change Maven dependency example
+displayName: "Update replaced maven artifact io.kroxylicious:kroxylicious-encryption with kroxylicious-record-encryption"
 recipeList:
   - org.openrewrite.maven.ChangeDependencyGroupIdAndArtifactId:
       oldGroupId: io.kroxylicious
       oldArtifactId: kroxylicious-encryption
       newGroupId: io.kroxylicious
       newArtifactId: kroxylicious-record-encryption
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: io.kroxylicious.rewrite.filterConfig.EnvelopeEncryptionFilterToRecordEncryptionFilter
+displayName: "Replace usages of EnvelopeEncryption Filter in config files with the RecordEncryption Filter"
+recipeList:
+  - org.openrewrite.yaml.ChangeValue:
```

Comment:
I'd like a separate recipe here along the lines of 
```yaml
  - org.openrewrite.yaml.ChangeValue:
      keyPath: $.data."config.yaml".filters.*.type[?(@ == 'EnvelopeEncryption')]
      value: RecordEncryption
```
To catch the values in the kubernetes-examples folder but the inner document is being considered as a single literal (I guess)

Reply:
It is not only the name of the factory that has been changed.  We've also made changes to io.kroxylicious.kms.provider.hashicorp.vault.config.Config which has affected the configuration.  We already have notes about this in the CHANGELOG.md.